### ISSUE TITLE
Could you provide detailed arguments on MSRaction3D?

### ISSUE THREAD

**Issue by car1ry** [JohnsonSign/MaST-Pre#6]
Thanks for releasing this great work.

 I am currently trying to reproduce the results on the MSR dataset, but the accuracy I obtain is about 10 percentage points lower than the one reported in the paper. I was wondering if you could share the exact training and fine-tuning parameters used for MSR.

 If possible, could you also provide the corresponding training logs and the args configuration file? That would help a lot in diagnosing where the discrepancy might come from

**Comment by JohnsonSign** [comment]
Sorry for the late reply. Has this issue been resolved now?
